App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/ContactDetail.jsx

```diff
@@ -19,6 +20,13 @@ export class IndividualDetailSection extends Component {
             showEditSection: false,
             newContact: details
         }
+        const phoneRegExp = /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/
```

Comment:
What is the difference between this reg exp and with line 171? If they are having the same purpose, then you can create probably a **util** folder and then set up something like **field-validation** file and then set this reg exp there and use it in this file. This way, you can be able to use single function anywhere in your project.

Reply:
Thanks Kester. I actually had a quick thought about it during development but didn't think of a util folder to place it. Now I took the time to think where to put it and found a util folder already in the repo, with a Talent-lib.js already used for notifications, so I moved the regex there for all project access.

Reply:
I have to admit though, I'm not too clear on where TalentUtil is being defined globally so that it is accessible in this component without any exports. Perhaps you can comment on that?